src/components/Collections/CollectionOptions/QuickOptions/ShareOption.tsx

```diff
@@ -0,0 +1,30 @@
+import { CollectionActions } from '..';
+import React from 'react';
+import PeopleIcon from '@mui/icons-material/People';
+import { IconButton, Tooltip } from '@mui/material';
+import constants from 'utils/strings/constants';
+
+interface Iprops {
+    handleCollectionAction: (
+        action: CollectionActions,
+        loader?: boolean
+    ) => (...args: any[]) => Promise<void>;
+    tooltipTitle?: string;
+}
+
+export function ShareOption({
```

Comment:
Following the convention of naming the component fully ( `ShareQuickOption` ) removes the confusion when importing it.

Else there can be cases where we can have `ShareCollectionOption` and `ShareQuickOption` both named as `ShareOption` and then will have to look into where the file is coming from to help figure out which one to import.

I know not a big issue... But just simply naming them completely fixes that import issue from ever happening. Also, `vscode` fix import suggestion will also never mistakenly import the wrong thing

Reply:
Ah, I think, you misunderstood this. I wanted the component to be named `ShareQuickOption`, but it's still called `shareOption` and you have renamed the filename to `ShareQuickOption`. Lets fix that